draft-bbs-signatures.md

```diff
@@ -273,11 +273,13 @@ As an example of the above transformations, consider the following. Assume that
 
 For readability, this document makes these transformations implicitly, but they MUST precede every call to the HASH or XOF function.
 
+Optional input/parameters to operations that feature in a call to a HASH or XOF function, that are not supplied to the operation should be omitted from the call, for example if X is an optional input/parameter whilst A and B are required then the procedural step of `HASH(A || X || B)` MUST be evaluated to `HASH(A || B)`.
```

Comment:
```suggestion
Optional input/parameters to operations that feature in a call to a HASH or XOF function, that are not supplied to the operation should default to the empty string. For example, if X is an optional, not supplied, input/parameter whilst A and B are required, then the procedural step of `HASH(A || X || B)` MUST be evaluated to `HASH(A || "" || B)`.
```
Since, in some places where we hash optional values we specify that if not supplied they default to the empty string, the above might be more accurate??

Reply:
What about empty octet string to be more specific?

Reply:
Yeap. That sounds better.